README.md

```diff
@@ -62,6 +63,7 @@ NOTE: You can get your API Key by signing up [here](https://www.validators.app/u
 21. [Polkadot](https://polkadot.network/)
 22. [Aptos](https://aptosfoundation.org/)
 23. [Sei](https://sei.io/)
+23. [Cardano](https://cardano.org/)
```

Comment:
@FilippoScaramuzza this is an alphabetical ordering. pls move this accordingly.

Reply:
I changed the chains order, so now it's alphabetical.